telemetry/app.go

```diff
@@ -0,0 +1,135 @@
+package telemetry
+
+import (
+	"context"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+
+	"cosmossdk.io/log"
+	storetypes "cosmossdk.io/store/types"
+	abci "github.com/cometbft/cometbft/abci/types"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/mempool"
+	"github.com/hashicorp/go-metrics"
+)
+
+const (
+	blockTxsSizeMetric    = "block_txs_size"
+	blockResultSizeMetric = "block_result_size"
+	diskUsageMetric       = "disk_usage"
```

Comment:
gb?

Reply:
Removed disk usage metric, since it would be a bottleneck to block production when the DB grows in size.